File: sprint5/C++/lesson2code1.cpp
Node* find_node(Node* root, int value) {
  if (root == nullptr) {
    return nullptr;
  }
  if (value < root->value) {
    return find_node(root->left, value);
  }
  if (value == root->value) {
    return root;
  }
  if (value > root->value) {
    return find_node(root->right, value);
  }
}

File: sprint5/Java/lesson2code1.java
public Node findNode(Node root, int value) {
  if (root == null) {
    return null;
  }
  if (value < root.getValue()) {
    return findNode(root.getLeft(), value);
  }
  if (value == root.getValue()) {
    return root;
  }
  if (value > root.getValue()) {
    return findNode(root.getRight(), value);
  }
}
